content/en/docs/collector/internal-telemetry.md

```diff
@@ -144,6 +144,62 @@ journalctl | grep otelcol | grep Error
 
 {{% /tab %}} {{< /tabpane >}}
 
+### Configure internal traces
+
+The Collector does not expose traces by default, but can be configured to.
+
+{{% alert title="Caution" color="warning" %}}
+
+Internal tracing is an experimental feature, and no guarantees are made as to
+the stability of the emitted span names and attributes.
+
+{{% /alert %}}
+
+The following configuration can be used to emit internal traces from the
+Collector to an OTLP/gRPC backend:
+
+```yaml
+service:
+  telemetry:
+    traces:
+      processors:
+        - batch:
+            exporter:
+              otlp:
+                protocol: grpc/protobuf
+                endpoint: https://backend:4317
+```
+
+See the [example configuration][kitchen-sink-config] for additional options (the
+`tracer_provider` section there corresponds to `traces` here).
+
+[kitchen-sink-config]:
+  https://github.com/open-telemetry/opentelemetry-configuration/blob/main/examples/kitchen-sink.yaml
+
+You can also configure the Collector to send its own traces using the OTLP
+exporter. Send the traces to an OTLP receiver running on the same Collector, so
+it goes through configured pipelines. For example:
+
+```yaml
+service:
+  telemetry:
+    traces:
+      processors:
+        batch:
+          exporter:
+            otlp:
+              protocol: grpc/protobuf
+              endpoint: localhost:4317
+```
+
+{{% alert title="Caution" color="warning" %}}
+
+Sending internal traces through the Collector's pipeline as above creates a
```

Comment:
Is there a reason to document this if it's not recommended?

Reply:
I mostly just copied the contents of the original document, but we can remove it if you think it's too problematic. I thought it would be alright since it's already documented and discouraged for metrics (*"The Collector can also be configured to push its own metrics to an OTLP receiver [...]"*).

Reply:
You're right, I'll consolidate the two warnings. (Although I believe the case for traces is more problematic than for metrics, since they are proactively exported (especially when not using the batch exporter), whereas metrics are just counted every 10s or so.)

Reply:
It's equally problematic for logs which will be configured shortly

Reply:
I haven't found what you're referring to (unless you mean the debugexporter?), but I'll update the warning to include a mention of logs as well.

Reply:
I believe Alex is talking about open-telemetry/opentelemetry-collector/pull/10544

Reply:
I reworded the warning about telemetry loops in a more generic way to include logs. I think adding an example of how to send internal logs through OTLP should be done in a separate PR once the version with that feature is released.